internal/signaling/stores/postgres.go

```diff
@@ -369,120 +369,184 @@ func (s *PostgresStore) ListLobbies(ctx context.Context, game, filter string) ([
 	return lobbies, nil
 }
 
-func (s *PostgresStore) TimeoutPeer(ctx context.Context, peerID, secret, gameID string, lobbies []string) error {
+func (s *PostgresStore) CreateOrUpdatePeer(ctx context.Context, peerID, secret string) error {
 	if len(peerID) > 20 {
 		logger := logging.GetLogger(ctx)
 		logger.Warn("peer id too long", zap.String("peerID", peerID))
 		return ErrInvalidPeerID
 	}
-	for _, lobby := range lobbies {
-		if len(lobby) > 20 {
-			logger := logging.GetLogger(ctx)
-			logger.Warn("lobby code too long", zap.String("lobbyCode", lobby))
-			return ErrInvalidLobbyCode
-		}
-	}
 
 	now := util.NowUTC(ctx)
 	_, err := s.DB.Exec(ctx, `
-		INSERT INTO timeouts (peer, secret, game, lobbies, created_at, updated_at)
-		VALUES ($1, $2, $3, $4, $5, $5)
+		INSERT INTO peers (peer, secret, created_at, updated_at)
+		VALUES ($1, $2, $3, $3)
 		ON CONFLICT (peer) DO UPDATE
 		SET
 			secret = $2,
-			game = $3,
-			lobbies = $4,
-			last_seen = $5,
-			updated_at = $5
-	`, peerID, secret, gameID, lobbies, now)
+			disconnected = FALSE,
+			last_seen = $3,
+			updated_at = $3
+	`, peerID, secret, now)
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
-func (s *PostgresStore) ReconnectPeer(ctx context.Context, peerID, secret, gameID string) (bool, []string, error) {
+func (s *PostgresStore) MarkAllPeersAsActive(ctx context.Context) error {
+	now := util.NowUTC(ctx)
+	_, err := s.DB.Exec(ctx, `
+		INSERT INTO peers (peer, last_seen, updated_at)
+		SELECT UNNEST(peers) AS peer, $1 AS last_seen, $1 AS updated_at
+		FROM lobbies
+		ON CONFLICT (peer)
+		DO UPDATE SET
+			last_seen = $1,
+			updated_at = $1
+	`, now)
```

Comment:
Is this still relevant? All peers are already created in the database right? Maybe just a update on the last_seen? (e.g. `UPDATE peers SET last_seen = NOW() WHERE 1`)